src/org/labkey/test/tests/SecurityTest.java

```diff
@@ -553,6 +555,23 @@ protected void clonePermissionsTest()
         assertNavTrail("Site Users", "User Details", "Permissions");
     }
 
+    protected void projectUsersTest()
+    {
+        beginAt("/project/SecurityVerifyProject/begin.view?");
+        impersonateRoles(PROJECT_ADMIN_ROLE);
+        ShowUsersPage usersPage = goToProjectUsers();
+        String userEmail = "fromprojectusers@gmail.com";
+
+        usersPage
+                .clickAddUsers()
+                .setNewUsers(Arrays.asList(userEmail))
+                .setSendNotification(true)
+                .clickAddUsers();
+
+        assertTextPresent(userEmail);
+        stopImpersonating();
+    }
+
```

Comment:
Could also add a check, if it seems worthwhile, to impersonate folder admin and verify the 'Add Users' button is not present.

Reply:
I think that's a worthwhile test. The distinction between project and folder admins can be unclear at times and would be worth verifying here.

Reply:
Done!